pkg/sql/parser/sql.y

```diff
@@ -5797,6 +5799,20 @@ show_transaction_stmt:
   }
 | SHOW TRANSACTION error // SHOW HELP: SHOW TRANSACTION
 
+// %Help: SHOW TRANSFER - display current transfer properties
```

Comment:
nit: "display information used by the sql proxy to transfer connections between sql instances"